neurom/fst/_neuronfunc.py

```diff
@@ -53,6 +53,16 @@ def soma_surface_area(nrn, neurite_type=NeuriteType.soma):
     return 4 * math.pi * nrn.soma.radius ** 2
 
 
+def soma_volume(nrn, neurite_type=NeuriteType.soma):
```

Comment:
Is the neurite_type argument mandatory ?

Reply:
yeah, that looks strange. It seems it is already like that in the other functions, so I assume @asanin-epfl is following the style.